src/apps/messages/entryPoints/messageRoutes.js

```diff
@@ -34,6 +35,9 @@ router.get(
   getMessageById,
 );
 
+//get messages by category
+router.get('/category/:id', getMessageByCategory);
```

Comment:
its better to add joi validation middle ware befor executing the controller function